new-docs/content/any/project/changelogs/NEXT.md

```diff
@@ -20,16 +35,24 @@ draft: true
   of instances are spun up and not cleanly reaped by the GC, such as during a
   long-running test process in 'watch' mode.
 
-### Rust
+- The Polar `Variable` type is now exposed in the Node.js library, allowing users to pass unbound variables to `queryRule()` and `isAllowed()`.
+```
+  const oso = new Oso()
+  await oso.loadStr('hello("world"); hello("something else");');
+  for await (const val of oso.queryRule("hello", new Variable("var"))) {
+      console.log(val);
+  }
+  => 
+  Map(1) { 'var' => 'world' }
+  Map(1) { 'var' => 'something else' }
+```
+
+### Go
 
 #### Other bugs & improvements
 
-- The Rust CLI now uses [`clap`](https://crates.io/crates/clap) to expose a
-  prettier interface thanks to
-  [@joshrotenberg](https://github.com/joshrotenberg) via [PR
-  #828](https://github.com/osohq/oso/pull/828).
+- Go lib no longer tries to print the zero values it uses for bookkeeping. This would crash when running on macOS under delve.
 
-### OTHER_LANGUAGE
```

Comment:
Why's this deleted?

Reply:
I dunno, because it's just a placeholder?

Reply:
Everything below it is a placeholder too 😄 

let's keep it for now